packages/aws-rfdk/lib/deadline/scripts/powershell/configureWorker.ps1

```diff
@@ -99,7 +99,9 @@ if($WORKER_POOLS) {
 
 $serviceName="deadline10launcherservice"
 If (Get-Service $serviceName -ErrorAction SilentlyContinue) {
-    Restart-Service $serviceName
+    Stop-Service $serviceName
+    taskkill /f /im deadlineworker.exe -ErrorAction SilentlyContinue
+    Start-Service $serviceName
 } Else {
     $DEADLINE_LAUNCHER = $DEADLINE_PATH + '/deadlinelauncher.exe'
     & $DEADLINE_LAUNCHER -shutdownall | Out-Default
```

Comment:
Just making sure we don't miss anything:
For Linux here we have sudo `killall -w deadlineworker || true`.

And on Windows we don't kill `deadlineworker` here. Could you please explain why?

Reply:
I have missed it here.